<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="all_tab">
       <property name="accessibleName">
        <string/>
       </property>
       <attribute name="title">
        <string>Allgemein</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,4,3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="2,1">
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>Wetter</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QGroupBox" name="groupBox_6">
                   <property name="title">
                    <string>Heute</string>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_7">
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_6" stretch="3,1">
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_9">
                        <item>
                         <widget class="QGroupBox" name="groupBox_7">
                          <property name="title">
                           <string>Vormittag</string>
                          </property>
                          <widget class="QLabel" name="wTodayImage1">
                           <property name="geometry">
                            <rect>
                             <x>17</x>
                             <y>25</y>
                             <width>64</width>
                             <height>46</height>
                            </rect>
                           </property>
                           <property name="text">
                            <string/>
                           </property>
                          </widget>
                         </widget>
                        </item>
                        <item>
                         <widget class="QGroupBox" name="groupBox_8">
                          <property name="title">
                           <string>Nachmittag</string>
                          </property>
                          <widget class="QLabel" name="wTodayImage2">
                           <property name="geometry">
                            <rect>
                             <x>17</x>
                             <y>25</y>
                             <width>64</width>
                             <height>46</height>
                            </rect>
                           </property>
                           <property name="text">
                            <string/>
                           </property>
                          </widget>
                         </widget>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <widget class="QLabel" name="wTodayText">
                        <property name="text">
                         <string>TextLabel</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="groupBox_5">
                   <property name="title">
                    <string>Morgen</string>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_9">
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_8" stretch="3,1">
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_10">
                        <item>
                         <widget class="QGroupBox" name="groupBox_9">
                          <property name="title">
                           <string>Vormittag</string>
                          </property>
                          <widget class="QLabel" name="wTomorrowImage1">
                           <property name="geometry">
                            <rect>
                             <x>17</x>
                             <y>25</y>
                             <width>64</width>
                             <height>46</height>
                            </rect>
                           </property>
                           <property name="text">
                            <string/>
                           </property>
                          </widget>
                         </widget>
                        </item>
                        <item>
                         <widget class="QGroupBox" name="groupBox_10">
                          <property name="title">
                           <string>Nachmittag</string>
                          </property>
                          <widget class="QLabel" name="wTomorrowImage2">
                           <property name="geometry">
                            <rect>
                             <x>17</x>
                             <y>25</y>
                             <width>64</width>
                             <height>46</height>
                            </rect>
                           </property>
                           <property name="text">
                            <string/>
                           </property>
                          </widget>
                         </widget>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <widget class="QLabel" name="wTomorrowText">
                        <property name="text">
                         <string>TextLabel</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="groupBox_11">
                   <property name="title">
                    <string>Ãœbermorgen</string>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_10">
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_11" stretch="3,1">
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_11">
                        <item>
                         <widget class="QGroupBox" name="groupBox_12">
                          <property name="title">
                           <string>Vormittag</string>
                          </property>
                          <widget class="QLabel" name="wOvermorrowImage1">
                           <property name="geometry">
                            <rect>
                             <x>17</x>
                             <y>25</y>
                             <width>64</width>
                             <height>46</height>
                            </rect>
                           </property>
                           <property name="text">
                            <string/>
                           </property>
                          </widget>
                         </widget>
                        </item>
                        <item>
                         <widget class="QGroupBox" name="groupBox_13">
                          <property name="title">
                           <string>Nachmittag</string>
                          </property>
                          <widget class="QLabel" name="wOvermorrowImage2">
                           <property name="geometry">
                            <rect>
                             <x>17</x>
                             <y>25</y>
                             <width>64</width>
                             <height>46</height>
                            </rect>
                           </property>
                           <property name="text">
                            <string/>
                           </property>
                          </widget>
                         </widget>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <widget class="QLabel" name="wOvermorrowText">
                        <property name="text">
                         <string>TextLabel</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Uhrzeit</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLCDNumber" name="clock"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Nachrichten</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QListWidget" name="RssList">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Termine</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QTableWidget" name="tableWidgetMain"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="wether_tab">
       <property name="accessibleName">
        <string>Wetter</string>
       </property>
       <attribute name="title">
        <string>Kalender</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTableWidget" name="TableCal">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
